projects/1.20/assets/game-discs/gamediscs/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+{
+    "creativetab.game_discs_tab": "游戏光碟",
+  
+  
+    "gamediscs.flappy_bird": "飞翔的蜜蜂",
+    "gamediscs.slime": "贪吃史莱姆",
+    "gamediscs.blocktris": "消除方块",
+    "gamediscs.tnt_sweeper": "清扫TNT",
+    "gamediscs.pong": "乒乓",
+    "gamediscs.froggie": "蛙跃水沟",
+    "gamediscs.pong_game": "乒乓",
+  
+  
+    "item.gamediscs.gaming_console": "游戏机",
+    "item.gamediscs.redstone_circuit": "红石电路",
+    "item.gamediscs.processor": "处理器",
+    "item.gamediscs.battery": "电池",
+    "item.gamediscs.display": "屏幕",
+    "item.gamediscs.control_pad": "控制器",
+    "item.gamediscs.game_disc_flappy_bird": "游戏光碟",
+    "item.gamediscs.game_disc_slime": "游戏光碟",
+    "item.gamediscs.game_disc_blocktris": "游戏光碟",
+    "item.gamediscs.game_disc_tnt_sweeper": "游戏光碟",
+    "item.gamediscs.game_disc_pong": "游戏光碟",
+    "item.gamediscs.game_disc_froggie": "游戏光碟",
+  
+  
+    "gui.gamingconsole.title": "游戏机",
+    "gui.gamingconsole.press_any_key": "请按下任意键",
+    "gui.gamingconsole.select_game": "选择游戏",
+    "gui.gamingconsole.score": "得分",
+    "gui.gamingconsole.best_score": "最佳",
+    "gui.gamingconsole.new_best_score": "新纪录",
+    "gui.gamingconsole.won": "你赢了！",
+    "gui.gamingconsole.died": "你输了！",
+    "gui.gamingconsole.hold": "持有",
+    "gui.gamingconsole.next": "下一个",
+  
+  
+    "sounds.gamediscs.jump": "游戏机：跳跃",
+    "sounds.gamediscs.click": "游戏机：按下按钮",
+    "sounds.gamediscs.point": "游戏机：目标",

```

这个音效是在什么情况下出现的

---

目前看来是游戏机的音量可配置项，但是作者可能还没做出来

---

这个应该是游戏机的[字幕](https://zh.minecraft.wiki/w/%E5%AD%97%E5%B9%95)，查证模组内`sound.json`确实如此
怀疑是得分时播放的音效，建议游戏内检验

---

经游戏内查证，Flappy Bee得分时会播放此音效
```suggestion
    "sounds.gamediscs.point": "游戏机：得分",
```